src/main/java/com/example/bookstoreapp/controller/ShoppingCartController.java

```diff
@@ -46,16 +48,16 @@ public ShoppingCartDto addItem(@RequestBody CartItemDto cartItemDto,
 
     @DeleteMapping("/cart-items/{id}")
     @PreAuthorize("hasRole('USER')")
-    @Operation(summary = "Delete item from shopping cart",
-            description = "Delete item from shopping cart")
+    @Operation(summary = "Delete from shopping cart",
+            description = "Delete item from current User's shopping cart")
     public void deleteCartItem(@PathVariable Long id) {
         shoppingCartService.deleteById(id);
```

Comment:
I do not see any validation that this cartItem belongs to the user who is making a request, let's add it, because from the looks of it, user can delete a cartItem that is not in his cart